qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -197,20 +212,33 @@ def _score_layout(self, layout):
         to weight against higher connectivity qubits."""
         bits = layout.get_physical_bits()
         score = 0
-        if self.properties is None:
-            # Sum qubit degree for each qubit in chosen layout as really rough estimate of error
+        if self.target is not None:
+            if "measure" in self.target:
+                for bit in bits:
+                    props = self.target["measure"].get((bit,))
+                    if props is None or props.error is None:
+                        score += (
+                            self.coupling_map.graph.out_degree(bit)
+                            + self.coupling_map.graph.in_degree(bit)
+                        ) / len(self.coupling_map.graph)
+                    else:
+                        score += props.error
+        else:
```

Comment:
I didn't notice this before because of the nesting - if `self.target is not None` but `measure not in self.target`, we just return a score of 0 for all layouts now.  I mean, I can't imagine a situation where `measure` _isn't_ in the target, but we may need to fall back to the "dumb" calculation?

Reply:
Yeah ideally I would have just fixed the heuristic to be more general like I did in #7776 for dense layout, but I didn't want to step on @1ucian0's toes as he had started working on a better heurstic in #7722. I'm generally not a fan of doing this with a hardcoded measurement lookup by name like this. I can update the logic really quickly to fix this gap

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/7735/commits/f7fb8a59968f465c1016f451e4a6462d2b920c8a